### ISSUE TITLE
php artisan clear-compiled generates error

### ISSUE THREAD

**Issue by yespbs** [laravel/framework#8520]
Created a new project from source today and clear compiled started to show error on composer update
What is missing?
-------------------------------
Generating autoload files
PHP Fatal error:  Call to undefined method Illuminate\Foundation\Application::getCachedCompilePath() in [path_to_install]/vendor/laravel/framework/src/Illuminate/Foundation/Console/ClearCompiledCommand.php on line 28

                                                                                      
  [Symfony\Component\Debug\Exception\FatalErrorException]                             
  Call to undefined method Illuminate\Foundation\Application::getCachedCompilePath()  
                                                                                      

Script php artisan clear-compiled handling the post-update-cmd event returned with an error

                                                                               
  [RuntimeException]                                                           
  Error Output: PHP Fatal error:  Call to undefined method Illuminate\Foundat  
  ion\Application::getCachedCompilePath() in [path_to_install]/vendor/laravel/framework/src/Illuminate/Foundation/  
  Console/ClearCompiledCommand.php on line 28

**Comment by GrahamCampbell** [comment]
You need to manually delete the compiled file. Look both in the storage folder and the vendor folder. :)

**Comment by yespbs** [comment]
ok thanks, so weird

**Comment by yespbs** [comment]
solved, thanks, first time issue with L5, got confused, thanks

**Comment by RSully** [comment]
This just happened to me with a clean/fresh install of v5. Is it still expected behavior to have to manually remove that file?

```
composer create-project laravel/laravel --prefer-dist my_project
cd my_project

composer update
# error
php laravel clear-compiled
# error

rm vendor/compiled.php

php laravel clear-compiled
# no error
composer update
# no error
```

**Comment by lukasgeiter** [comment]
@RSully AFAIK this has been completely fixed in *master* and will be in the 5.1 release.

**Comment by GrahamCampbell** [comment]
See https://github.com/laravel/framework/issues/9678.
